docs/configuration/observability.md

```
@@ -73,6 +73,104 @@ export ARGOCD_AGENT_LOG_LEVEL=info
+argocd-agent agent --full-detail=all
+```
+
+Via environment variable:
+
+```bash
+export ARGOCD_AGENT_FULL_DETAIL="actions,events"
+```
+
+**Example output with `--full-detail=actions`:**
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)